runtime/src/main/codestarts/quarkus/openapi-generator-codestart/java/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+quarkus:
```

Comment:
I'm wondering if we shouldn't use application.properties instead. I'm worried about mixing formats (from other extensions). Doesn't codestarts have a standard?

Reply:
This is the standard of codestarts, at the end, the CLI will merge all properties into the application.properties file. See here https://quarkus.io/guides/extension-codestart#app-config